components/Buttons/UserButton.vue

```diff
@@ -42,14 +38,32 @@ export default Vue.extend({
   },
   computed: {
     menuItems(): MenuItem[] {
-      return [
-        {
-          title: this.$t('logout'),
+      const menuItems = [];
+
+      if (this.$auth.$state.user.Policy.IsAdministrator) {
+        menuItems.push({
```

Comment:
I don't think metadata fits here, wouldn't it be better to move it to a place within settings?

Reply:
I know a few of the users want it accessible without drilling down into pages. A metadata editor isn't really a setting either :/

This felt like a decent place for it, so it's out of the way of the main UI (It's not something you use everyday) but is still accessible from everywhere.

Reply:
I don't mind either, but I think another good place for it would be alongside the top bar icons, while in home.